src/main/interface/settings.c

```diff
@@ -1013,8 +1046,12 @@ const clivalue_t valueTable[] = {
 // PG_LED_STRIP_CONFIG
 #ifdef USE_LED_STRIP
     { "ledstrip_visual_beeper",     VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_LED_STRIP_CONFIG, offsetof(ledStripConfig_t, ledstrip_visual_beeper) },
+#ifdef USE_LED_STRIP_STATUS_MODE
```

Comment:
`ledstrip_grb_rgb` is applied in the LED_STRIP driver, so it should be available even without `USE_LED_STRIP_STATUS_MODE`.

Reply:
Done.